index.html

```diff
@@ -70,6 +70,41 @@
     <li><button class='arrowDirectionDialog'>Arrow Direction Dialog</button></li>
```

Comment:
This example isn't working well, I assume because the anchor and dialog it opens aren't offset parents.